code/game/objects/items/clown_items.dm

```diff
@@ -125,6 +124,10 @@
 	honksounds = list('sound/items/airhorn2.ogg'=1)
 	origin_tech = "materials=4;engineering=4"
 
+/obj/item/bikehorn/airhorn/Initialize()
+	. = ..()
+	AddComponent(/datum/component/squeak, list('sound/items/airhorn2.ogg'=1), 50)

```

Comment:
So, you know this will delete the component created by /obj/item/bikehorn, right?

Reply:
yeah, it's meant to override the old one for a new sound